### PR TITLE
Updates contribex meetings in sigs.yaml

### LINKED ISSUE #4111: contribex meetings updated in sigs.yaml
need to have the mentoring and contributor documentation meetings updated with times/dates/other info in sigs.yaml (in the root of kubernetes/community)

also need to remove the meeting information for the noncode guide since that is flowing into the contributor documentation meeting

contribex meeting agenda has all of the above information in detail on the first page
[meeting notes](https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/edit?usp=sharing)

update meetings that need to be updated and their times:
triage meeting (under the biweekly meeting): bi weekly 1030a-1130a Weds https://zoom.us/j/712527529
apac meeting (community management subproject) 9-930pm PT bi weekly wednesdays
https://zoom.us/j/712527529
upstream marketing (community management subproject) fridays @ 8am-9am PT 
https://zoom.us/j/596959769

### PR DESCRIPTION
**Pull request by SomtochiAma**
**Which issue(s) this PR fixes**:

Fixes #4111

### CODE REVIEW

File: sig-list.md [hunk 1]
```diff
@@ -30,7 +30,11 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
+<<<<<<< HEAD
```
Comment (cblecker) [inline sig-list.md]: The merge conflict here isn't properly resolved. Please rebase your PR.

**Comment by SomtochiAma** [comment]
> @SomtochiAma Would you mind squashing your commits down and updating the commit message to be something like "Update sig contribex meetings"?
> After that things should be good. 👍

Thank you. I have squashed the commits

**Comment by mrbobbytables** [comment]
It looks like you wiped out the generated files 😬 
Also the commit message is currently `Updates contributex meetings in sigs.yaml`
It should be `Update sig contribex meetings`

**Comment by SomtochiAma** [comment]
> It looks like you wiped out the generated files 😬
> Also the commit message is currently `Updates contributex meetings in sigs.yaml`
> It should be `Update sig contribex meetings`

Sorry! I have made the corrections

**Comment by mrbobbytables** [comment]
Another PR landed, so this will need to be rebased. 😬 The comment message should also be:

Updates sig **contribex** meetings

👍

**Comment by mrbobbytables** [comment]
It'll need to be rebased again >_< There's been a lot of activity in there lately which is throwing it off =/ Sorry I didn't get to it earlier.

**Comment by SomtochiAma** [comment]
> It'll need to be rebased again >_< There's been a lot of activity in there lately which is throwing it off =/ Sorry I didn't get to it earlier.

It is all good. Pushing now.

**Comment by parispittman** [comment]
/lgtm

Thank you so much! Reach out to me on k8s slack ‘paris’ and I can get you a code to the online swag store if you want some :)

**Pull request merged**